.github/actions/setup-database/action.yml

```diff
@@ -0,0 +1,11 @@
+name: "Set up database"
+description: "Create db and run migrations"
+runs:
+  using: "composite"
```

Comment:
[question] What makes this a composite action?  It's my understanding from very cursory searching around that composite github actions mean multiple steps in the same action, but in this case it loos like there is just one step:  set up test database

Reply:
The fact that it's not a "docker" or "javascript" action. Also, this one is now multiple run steps because why not.